### ISSUE TITLE
stt.exe keeps closing

### ISSUE THREAD

**Issue by marL0man** [KoljaB/RealtimeSTT#146]
I keep opening the exe file for stt, and it only ever works for a couple seconds. It opens fine and connects, but it always shuts itself down after one or two seconds.

Video:
https://youtu.be/PBlQYPvapDc

**Comment by KoljaB** [comment]
This is probably the expected behavior when starting the exe. The STT client connects, waits for a speech, displays the transcription and terminates after voice deactivity. I assume that if you start it by clicking the .exe it just closes then. 
What does it do if you open a terminal and type STT there? You can try "STT -D" for extended debugging to see what's going on. Or try "STT -l" (for loop) and it should not terminate after voice deactivity anymore. 
Long story short: it's not made for starting via clicking the exe.

**Comment by marL0man** [comment]
oh yeah, my mistake, sorry
